.github/workflows/build-and-release-ctl.yml

```diff
@@ -28,16 +27,42 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_NOTES_PAT }}
           RELEASE_CHANNEL: "stable"
-          Tag: ${{inputs.release-tag}}
+          TAG: ${{inputs.release-tag}}
         with:
           version: 2
           args: release --clean --skip=validate -f .github/.goreleaser.yml
+
+        # ---------------- Cosign signing step----------------
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.5.0
+
+      - name: Sign mesheryctl artifacts (keyless)
+        run: |
+          for artifact in dist/*; do
+            cosign sign-blob \
+              --yes \
+              --bundle "${artifact}.sigstore" \
+              --output-signature "${artifact}.sig" \
+              --output-certificate "${artifact}.pem" \
```

Comment:
do we need the --output - the docs here https://docs.sigstore.dev/cosign/signing/signing_with_blobs/ suggest dont have to separate for specify sign and cert